recipes/multicolor-text/demos/rainbow/index.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html lang="ru">
+<head>
+  <title>Текст с фоном-радугой. Финальный результат — Разноцветный текст — Дока</title>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Roboto&display=swap">
+  <style>
+    body {
+      margin: 0;
+      padding: 50px;
+      background-color: #18191c;
+      font-family: "Roboto", sans-serif;
+      height: 100%;
+      min-height: 340px;
+      display: flex;
+      justify-content: center;
+      align-items: center;
+    }
+
+    h1 {
+      background-image: linear-gradient(250deg, rgba(131,58,180,1) 0%, rgba(253,29,29,1) 50%, rgba(252,176,69,1) 100%);
```

Comment:
```suggestion
      background-image: linear-gradient(250deg, rgb(131 58 180) 0%, rgb(253 29 29) 50%, rgba(252 176 69) 100%);
```

Мы используем везде новый синтаксис RGB. Кстати, тут все цвета на 100%. Точно нужен RGB или можно заменить на HEX?

Reply:
Коммент релевантен для кода ниже